.github/workflows/mantle-releases-main.yml

```diff
@@ -54,7 +54,7 @@ jobs:
         id: fetch-latest-mantle
         run: |
           commit=$(git ls-remote  https://github.com/flatcar/mantle refs/heads/flatcar-master | cut -f1)
-          echo ::set-output name=COMMIT::${commit}
+          echo "COMMIT=${COMMIT}" >>"${GITHUB_OUTPUT}"
```

Comment:
Maybe this should be `echo "COMMIT=${commit}" >>"${GITHUB_OUTPUT}"`

Reply:
But it's also good to add a check later for empty `COMMIT`

Reply:
Yeah, that should have been `"${commit}"`.

Reply:
I'll make the change and will test it.

Reply:
Thanks